enterprise/backend/src/metabase_enterprise/metabot_v3/dummy_tools.clj

```diff
@@ -24,6 +27,73 @@
                {:error "dashboard not found"})
    :context context})
 
+(defn- convert-field-type
+  [db-field]
+  (let [db-field (u/normalize-map db-field)]
+    (cond
+      (lib.types.isa/boolean? db-field)               "boolean"
+      (lib.types.isa/string-or-string-like? db-field) "string"
+      (lib.types.isa/numeric? db-field)               "number"
+      (lib.types.isa/temporal? db-field)              "date")))
+
+(defn- convert-field
+  [db-field]
+  (-> db-field
+      (select-keys [:id :name :description])
+      (update :id #(str "field_" %))
```

Comment:
These IDs can look somewhat strange for models, and they are not global like the ones for tables.

Reply:
**question:**
Do you have an example of how this would look like for models?

FWIW: The goal is that if we get the value of this `field_id` in a tool call from the LLM, we can 100% reliably determine which field it refers to. 

I was thinking about 3 cases here (likely I am missing something):
1. The field refers to a field in a **database table** -> ✅  should be easy since the table fields are unique (likely no need for the `field_` prefix and we can just pass the integer as a string)
2. The field refers to a field in a **query result** -> ☑️ (?) Here I was thinking if we have `query_id` + `field_id` (which could be something like `aggregation_0` or some other field ref we use in the MBQL query), we should be able to work with the tool call
3. The field refers to a field in a model -> ?? Here my understanding is that they are not unique per se but only in the combination with the model. Assuming the ids are something like `created_at` then `<model_id>__created_at` would make it unique. So if we get `123__created_at` from a tool call, we should be able to deconstruct this.

Reply:
If the field ID has to be a global ID (i.e., useable on its own, without knowing what model we are talking about), then the model's ID have to be added to this field.

This is how  a "strange" ID currently looks like:
`{"id":"field_[:field \"avg\" {:base-type :type/Float}]","name":"avg","type":"number"}`. To make it global, the model ID has to be added to it.